slices/login.yaml

```diff
@@ -0,0 +1,32 @@
+package: login
+
+essential:
+  - login_copyright
+
+slices:
+  bins:
+    essential:
+      - libaudit1_libs
+      - libc6_libs
+      - libcrypt1_libs
+      - libpam-modules_libs
+      - libpam-runtime_config
+      - libpam0g_libs
+      - login_config
+    contents:
+      /usr/bin/faillog:
+      /usr/bin/lastlog:
+      /usr/bin/login:
+      /usr/bin/newgrp:
+      /usr/bin/sg:
+      /usr/sbin/nologin:
+      /usr/share/apport/package-hooks/source_shadow.py:
```

Comment:
need testing.

The last one is a python script. Is any of these dependencies bringing in python?

Reply:
removing that python script because it is a logging script that doesn't seem to be called, and it depends on apport and python which are not referenced in the dependencies.